frontend/src/metabase/static-viz/components/ScatterPlot/ScatterPlot.tsx

```diff
@@ -0,0 +1,60 @@
+import { init } from "echarts";
+import { Group } from "@visx/group";
+
+import { getCartesianChartModel } from "metabase/visualizations/echarts/cartesian/model";
+import { getCartesianChartOption } from "metabase/visualizations/echarts/cartesian/option";
+import { sanitizeSvgForBatik } from "metabase/static-viz/lib/svg";
+import type { IsomorphicStaticChartProps } from "metabase/static-viz/containers/IsomorphicStaticChart/types";
+import { getLegendItems } from "metabase/visualizations/echarts/cartesian/model/legend";
+
+import { calculateLegendRows } from "../Legend/utils";
+import { Legend } from "../Legend";
+import { computeStaticComboChartSettings } from "../ComboChart/settings";
+
+const WIDTH = 540;
+const HEIGHT = 360;
+const LEGEND_PADDING = 8;
+
+export function ScatterPlot({
```

Comment:
This file fully duplicates the ComboChart. With the existing implementation the static `ComboChart` can be renamed to `CartesianChart` used for the scatter one too. But if you have plans to have separate models, wdyt about extracting from here the shared part so we don't duplicate a lot?